### PR TITLE
RPD - Clicking unwrenched pipes deletes them, clicking paintable pipes paints them

### PR DESCRIPTION
**Pull request by cacogen**
:cl: cacogen
tweak: RPD - No separate paint/eat modes anymore. Clicking unwrenched pipes deletes them, clicking paintable pipes paints them.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

The RPD's UI is very clunky and this reduces that clunkiness slightly by removing two of the modes.

- [x] Hitting unsecured pipe with RPD eats it regardless of mode
- [x] Hitting secured, paintable pipe with RPD paints it regardless of mode
- [x] Remove paint/eat modes entirely

### CODE REVIEW

**Comment by duncathan** [comment]
I really, really don't like this. This will make large projects, especially multipipe ones, a huge pain in the ass to create since I'll have to be way more precise with my clicking. If there were some sort of modifier key for this or an action button that would cycle through the modes without using the RPD UI I would be much happier and it'd still speed up usage. The action button is probably the easiest method.

As is though, I will not merge this PR.

**Review by duncathan — CHANGES_REQUESTED** [review]
^

**Comment by cacogen** [comment]
Atoms pipes are quite small. There’s a 500ms delay to prevent accidents too. Download it and test it out.

The problem I have with cycling through modes is when it’s laggy it’s easy to overshoot your mark and find yourself having to do two or three more cycles to get to the setting that you want. I have this trouble with the RCD. That and it’s still fairly clunky.

**Comment by nicbn** [comment]
I'll update the tgui and PR into your branch, hold on

**Comment by nicbn** [comment]
Actually I'm going to PR it separatedly later

**Comment by nicbn** [comment]
This closes and is closed by #36417

**Comment by nicbn** [comment]
My PR is ready on your branch

**Comment by DaxDupont** [comment]
I like #36417 far more

**Comment by cacogen** [comment]
If it does all the stuff this does and more I'd prefer it too

**Comment by nicbn** [comment]
This actually doesn't close the other PR, both features can be implemented

File: code/game/objects/items/RPD.dm [hunk 1]
```diff
@@ -70,6 +68,21 @@ GLOBAL_LIST_INIT(transit_tube_recipes, list(
 ))
 
 
+GLOBAL_LIST_INIT(pipe_paint_colors, list(
```
Comment (ShizCalev) [inline code/game/objects/items/RPD.dm]: These are used in more places than just this file, please keep it in the defines.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by ninjanomnom** [comment]
Ping for reopen when you start work on this again.

**Pull request closed without merging**